Source/JavaScriptCore/dfg/DFGOperations.cpp

```diff
@@ -4081,6 +4081,12 @@ JSC_DEFINE_JIT_OPERATION(operationDateGetYear, EncodedJSValue, (VM* vmPointer, D
     return JSValue::encode(jsNumber(gregorianDateTime->year() - 1900));
 }
 
+JSC_DEFINE_JIT_OPERATION(operationDateSetTime, EncodedJSValue, (VM*, DateInstance*, double time))
+{
+    double value = timeClip(time);
+    return JSValue::encode(jsDoubleNumber(value));
+}
+
```

Comment:
Why is this needed?  I don't see it called anywhere.

Reply:
Oops, removed.